packages/vaadin-grid/src/vaadin-grid-helper.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+/**
+ * @param {Array<Object>} columns array of columns to be modified
+ * @param {number} scope multiplier added to base order for each column
+ * @param {number} baseOrder base number used for order
+ */
+export function updateColumnOrders(columns, scope, baseOrder) {
+  let c = 0;
+  columns.forEach((column, _) => {
+    // avoid multiples of 10 because they introduce and extra zero and
```

Comment:
I've got the feeling the comment doesn't really match the code or am I missing something? c mod 9 === 0 would only trigger with 9,18,27... but not with 10,20,30 because those would return 1,2,3 and so on.. resulting in adding c+ (9->10)?

Reply:
Yeah, make sense, I'll have a look at it.